src/include/network/postgres/statement.h

```diff
@@ -141,7 +155,7 @@ class Statement {
   // The following objects can be "cached" in Statement objects for future statement invocations. Though they don't
   // relate to the Postgres Statement concept, these objects should be compatible with future queries that match the
   // same query text. The exception to this that DDL changes can break these cached objects.
-  std::unique_ptr<planner::AbstractPlanNode> physical_plan_ = nullptr;                // generated in the Bind phase
+  std::unique_ptr<optimizer::OptimizeResult> optimize_result_ = nullptr;              // generated in the Optimize phase
```

Comment:
This should still be Bind. There's no such thing as an Optimize phase. Postgres protocol supports Parse, Bind, Execute.